CenterForOpenScience/osf.io
# Feature/fix google drive path logging [ENG-692]

## Purpose


Ensure that the entire path of a folder is not displayed in node logs

## Changes


The `googledrive_folder_selected` NodeLog objects’ `params.folder` attribute includes the path of the folder. This attribute is used to generate the log message for the `googledrive_folder_selected` action. I modified the javascript code in *logTextParser.js* to use the `params.folder_name` attribute which does not include the folder path. 

## QA Notes


Yes, this change should be tested by QA to ensure that there are no situations in which the whole Google Drive path is exposed. Project overview pages (along with other project pages if they display node logs) should be tested with a variety of shared Google Drive folders/paths. 

## Documentation


N/A

## Side Effects


N/A
## Ticket


https://openscience.atlassian.net/browse/ENG-692
